src/stuf/stuf_zds/templates/stuf_zds/soap/creeerZaak.xml

```diff
@@ -101,10 +101,12 @@
                         </ZKN:nietNatuurlijkPersoon>
                     {% elif initiator.handelsnaam %}
                         <ZKN:nietNatuurlijkPersoon StUF:entiteittype="NNP" StUF:verwerkingssoort="T">
-                            <BG:statutaireNaam>{{ initiator.handelsnaam }}</BG:statutaireNaam>
                             <BG:authentiek>N</BG:authentiek>
+                            <BG:statutaireNaam>{{ initiator.handelsnaam }}</BG:statutaireNaam>
```

Comment:
ZKN:natuurlijkPersoon has `<BG:authentiek StUF:metagegeven="true">N</BG:authentiek>` while this line is missing the `StUF:metagegeven="true"` - why is this different?

@joeribekker do you know which it should be?